internal/services/springcloud/spring_cloud_elastic_application_performance_monitoring_resource_test.go

```diff
@@ -0,0 +1,168 @@
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (r SpringCloudElasticApplicationPerformanceMonitoringResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := appplatform.ParseApmID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.AppPlatform.AppPlatformClient.ApmsGet(ctx, *id)
+	if err != nil {
+		if response.WasNotFound(resp.HttpResponse) {
+			return pointer.To(false), nil
+		}
+		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
+	}
+	return pointer.To(true), nil
+}
+
+func (r SpringCloudElasticApplicationPerformanceMonitoringResource) template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-spring-%[2]d"
+  location = "%[1]s"
+}
+
+resource "azurerm_spring_cloud_service" "test" {
+  name                = "acctest-sc-%[2]d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  sku_name            = "E0"
+}
+`, data.Locations.Primary, data.RandomInteger)
+}
+
+func (r SpringCloudElasticApplicationPerformanceMonitoringResource) basic(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_spring_cloud_elastic_application_performance_monitoring" "test" {
+  name                    = "acctest-apm-%[2]d"
+  spring_cloud_service_id = azurerm_spring_cloud_service.test.id
+  application_packages    = ["org.example", "org.another.example"]
+  service_name            = "test-service-name"
+  server_url              = "http://127.0.0.1:8200"
+}
+`, template, data.RandomInteger)
+}
+
+func (r SpringCloudElasticApplicationPerformanceMonitoringResource) requiresImport(data acceptance.TestData) string {
+	config := r.basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_spring_cloud_elastic_application_performance_monitoring" "import" {
+  name                    = azurerm_spring_cloud_elastic_application_performance_monitoring.test.name
+  spring_cloud_service_id = azurerm_spring_cloud_elastic_application_performance_monitoring.test.spring_cloud_service_id
+  application_packages    = ["org.example", "org.another.example"]
+  service_name            = "test-service-name"
+  server_url              = "http://127.0.0.1:8200"
+}
+`, config)
+}
+
+func (r SpringCloudElasticApplicationPerformanceMonitoringResource) complete(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_spring_cloud_elastic_application_performance_monitoring" "test" {
+  name                    = "acctest-apm-%[2]d"
+  spring_cloud_service_id = azurerm_spring_cloud_service.test.id
+  application_packages    = ["org.example", "org.another.example"]
+  service_name            = "test-service-name"
+  server_url              = "http://127.0.0.1:8200"
```

Comment:
These values are identical to the ones in the `basic` config but they can be updated, can you please change them so we can test the update path for these too?